### PR TITLE
survivalgames

### PR DESCRIPTION
**Pull request by koisio**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/survivalgames/manifest.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "direct_ip": "mc.survival-games.cz",
+  "supported_languages": [
+     "cs",
+     "sk"
+	 "en"
+  ],
+  "social": {
+    "web": "https://survival-games.cz",
+    "web_shop": "https://store.survival-games.cz",
+    "web_support": "https://account.survival-games.cz",
```
Comment (maaaathis) [inline minecraft_servers/survivalgames/manifest.json:13]: Cannot be a login protected page; use a public faq or similar

File: minecraft_servers/survivalgames/manifest.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "facebook": "CZSurvivalGames", 
+    "instagram": "survival.games.cz",
+    "youtube": "https://www.youtube.com/@SurvivalGamesCZE"
+	"discord": "https://discord.com/invite/ZvK9g2WaVs"
+	"teamspeak": "ts3.survival-games.cz"
+	"tiktok": "https://www.tiktok.com/@survivalgamescz"
+  },
+  "discord": {
+    "server_id": 694511728691511347,
+    "rename_to_minecraft_name": true
```
Comment (maaaathis) [inline minecraft_servers/survivalgames/manifest.json:23]: Reserved for partners only; change to `false`

File: minecraft_servers/survivalgames/manifest.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+  "server_name": "survivalgames",
+  "nice_name": "SurvivalGames",
+  "direct_ip": "mc.survival-games.cz",
+  "supported_languages": [
+     "cs",
+     "sk"
+	 "en"
+  ],
```
Comment (maaaathis) [inline minecraft_servers/survivalgames/manifest.json:9]:
```suggestion
  "supported_languages": [
     "cs",
     "sk"
     "en"
  ],
```

File: minecraft_servers/survivalgames/manifest.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  "social": {
+    "web": "https://survival-games.cz",
+    "web_shop": "https://store.survival-games.cz",
+    "web_support": "https://account.survival-games.cz",
+    "facebook": "CZSurvivalGames", 
+    "instagram": "survival.games.cz",
+    "youtube": "https://www.youtube.com/@SurvivalGamesCZE"
+	"discord": "https://discord.com/invite/ZvK9g2WaVs"
+	"teamspeak": "ts3.survival-games.cz"
+	"tiktok": "https://www.tiktok.com/@survivalgamescz"
```
Comment (maaaathis) [inline minecraft_servers/survivalgames/manifest.json:19]:
```suggestion
    "facebook": "CZSurvivalGames", 
    "instagram": "survival.games.cz",
    "youtube": "https://www.youtube.com/@SurvivalGamesCZE"
    "discord": "https://discord.com/invite/ZvK9g2WaVs"
    "teamspeak": "ts3.survival-games.cz"
    "tiktok": "https://www.tiktok.com/@survivalgamescz"
```

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Comment by maaaathis** [comment]
Image pipeline fails: `Error: Could not read image file: minecraft_servers/survivalgames/background@2x.jpg`

File: minecraft_servers/survivalgames/manifest.json (hunk 1 above)
Comment (koisio) [reply minecraft_servers/survivalgames/manifest.json:13]: Okay i will ad the FAQ page.

File: minecraft_servers/survivalgames/manifest.json (hunk 2 above)
Comment (koisio) [reply minecraft_servers/survivalgames/manifest.json]: I changed it now

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by koisio** [comment]
Hello, is Survival Games ready to be added?
Thank you for your response,
Koisio

File: minecraft_servers/survivalgames/manifest.json [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  "social": {
+    "web": "https://survival-games.cz",
+    "web_shop": "https://store.survival-games.cz",
+    "web_support": "https://survival-games.cz/faq",
+    "facebook": "CZSurvivalGames", 
+    "instagram": "survival.games.cz",
+    "youtube": "https://www.youtube.com/@SurvivalGamesCZE",
+    "discord": "https://discord.com/invite/ZvK9g2WaVs",
+    "teamspeak": "ts3.survival-games.cz",
+    "tiktok": "https://www.tiktok.com/@survivalgamescz"
```
[the remark this answers is not in the document]
Comment (koisio) [reply minecraft_servers/survivalgames/manifest.json:23]: Alright thank you

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**